libraries/AP_Relay/AP_Relay.h

```diff
@@ -73,6 +73,8 @@ class AP_Relay {
     // see if the relay is enabled
     bool enabled(AP_Relay_Params::FUNCTION function) const;
 
+    int8_t get_gpio_pin(uint8_t instance) const { return _params[instance].pin.get(); }
```

Comment:
I still think this should just be `get_pin` (like in the leak detector code), and it's diabolical that there's an existing `get_pin` function that actually gets the state of the relay output assigned to a specified pin (instead of being called `get_state(pin)`, but anyway, those can be dealt with in the forward-port of this functionality to master...

Reply:
#31666